assets/src/edit-story/components/library/panes/pageTemplates/pageTemplatesPane.js

```diff
@@ -85,8 +90,13 @@ function PageTemplatesPane(props) {
   );
 
   const loadTemplates = useCallback(() => {
-    getCustomPageTemplates().then(setSavedTemplates);
-  }, [getCustomPageTemplates, setSavedTemplates]);
+    getCustomPageTemplates().then(({ templates, hasMore }) => {
+      setSavedTemplates(templates);
+      if (!hasMore) {
+        setNextTemplatesToFetch(false);
```

Comment:
I personally feel that -1 or 0 would be a better value for "no more pages to fetch" rather than switching from number to boolean. But that's a nit!